zebra-consensus/src/chain.rs

```diff
@@ -170,6 +167,13 @@ where
                     .await?;
             }
 
+            tracing::trace!(?height, ?hash, "Verified block");
+            metrics::gauge!(
+                "chain.verified.block.height",
+                height.expect("Valid blocks have a block height").0 as _
+            );
+            metrics::counter!("chain.verified.block.count", 1);
+
```

Comment:
Actually, I'll want to delete this in #969, so I'd best do it now.
```suggestion
```

Reply:
Hmm actually no I want to keep this, so we have a single metric which covers checkpoints and blocks.